docker/incremental_load.sh.tpl

```diff
@@ -72,6 +72,21 @@ function tag_layer() {
   fi
 }
 
+function read_variables() {
+  local file=${RUNFILES}/$1
+  local new_file=$(mktemp)
+
+  # Rewrite the file from Bazel for the form FOO=...
+  # to a form suitable for sourcing into bash to expose
+  # these variables as substitutions in the tag statements.
+  sed -E "s/^([^ ]+) (.*)\$/export \\1='\\2'/g" < ${file} > ${new_file}
+  source ${new_file}
+  rm ${new_file}
```

Comment:
Add a trap instead so a failure remove the file?

`trap "rm -f ${new_file}" EXIT`

(side-note: the bash magic avoids the creation of a temporary file, that's why I proposed it).